src/components/modules/banners/ausMoment/components/AusBannerCtas.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { css } from '@emotion/core';
+import { ThemeProvider } from '@emotion/react';
+import { space } from '@guardian/src-foundations';
+import { LinkButton, buttonReaderRevenue, buttonBrand } from '@guardian/src-button';
+import { neutral } from '@guardian/src-foundations/palette';
+import { Inline, Hide } from '@guardian/src-layout';
+import { BannerTextContent } from '../../common/types';
+import { SecondaryCtaType } from '../../../../../types/shared';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import { Link, linkBrand } from '@guardian/src-link';
+
+const containerStyles = css`
+    > * {
+        margin-bottom: 0;
+    }
+    align-items: center;
+`;
+
+const primaryCtaStyles = css`
+    color: ${neutral[0]};
+
+    &:hover {
+        color: ${neutral[100]};
+        background-color: ${neutral[0]};
+    }
+`;
+
+const secondaryCtaStyles = css`
+    margin-left: ${space[12]}px;
+
+    &:hover {
+        background-color: ${neutral[0]};
```

Comment:
should the hover state background be black?

Reply:
Good spot! I've changed it to reverse colours